lib/locales/en.ts

```diff
@@ -639,15 +648,18 @@ export const locale: Locale = {
     luigi_orione_priest: 'Saint Luigi Orione, Priest',
     luke_evangelist: 'Saint Luke, Evangelist',
     lydia_of_philippi: 'Saint Lydia of Philippi',
+    lydia_of_philippi_first_european_christian: 'Saint Lydia of Philippi, the first Christian of Europe',
```

Comment:
Actually, `lydia_of_philippi` is inscribed in the Greek National Proper only (no other calendar in romcal has that celebration inscribed).

Therefore, I think it it safe to replace `lydia_of_philippi` with `lydia_of_philippi_first_european_christian` in every locale and `greek.ts`. Moreover, it would be nice to add `FIRST_EUROPEAN_CHRISTIAN` (or `FIRST_CHRISTIAN` with place set to `EUROPE`) to the list of patronage titles.

Reply:
`lydia_of_philippi_first_european_christian` should be renamed to `lydia_of_philippi_the_first_european_christian`.

The question is, however, whether we should also rename it in `martyrology.ts` or only add `martyrology: ['lydia_of_philippi']` in the celebration definition in `greece.ts`. :thinking:

I am about to rename it everywhere (incl `martyrology.ts`) to `lydia_of_philippi_the_first_european_christian`.